### ISSUE TITLE
Enhancement: SVG separators/borders should not be visible to assistive technologies

### ISSUE THREAD

**Issue by nicolasgalvez** [stellarwp/kadence-blocks#641]
Hi folks,

Adding an aria-role tag to the kt-row-layout-* svg elements would meet [WCAG 1.1.1]( https://www.w3.org/WAI/WCAG21/Understanding/non-text-content)

I was working on a site where the element `.kt-row-sep-type-waves svg` failed validation. "This svg element is purely decorative, but without properly hiding this component, some assistive technologies may treat the SVG as content that requires a description."

I tested this in VoiceOver on Chrome, and the image was picked up by the screen reader when using the right and left arrow keys. I don't think there is a downside to adding an aria-role tag to the separator SVGs, it seems like an easy fix so I'll make a PR.

Thanks for the awesome theme and plugin!
